File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
import java.util.*;

class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        int flag = 1;
        if(a > b)
            flag *= -1;
        for(int i = 0; i <= Math.abs(a - b); i++){
            answer += (long)a + i * flag;
        }
        return answer;
    }
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.cpp
#include<bits/stdc++.h>

using namespace std;

long long solution(int a, int b) {
    long long answer = 0;
    int flag = 1;
    if(a > b)
        flag *= -1;
    for(int i = 0; i <= abs(a - b); i++){
        answer += a + i * flag ;
    }
    return answer;
}